src/components/Node/Node.js

```diff
@@ -424,6 +417,7 @@ const ThemedNodeOuter = styled.div`
   ${({ theme }) => theme.nodeFlexDirection === "row" && `align-items: center`};
   ${({ theme }) =>
     theme.nodeFlexDirection === "column" && `justify-content: center`};
+  ${({ theme }) => theme.customCss};
```

Comment:
I did this on the other PR as well, I think `nodeCss` is more appropriate, they are all custom

Reply:
Ok I'll merge it